selenium/src/test/java/au/org/emii/portal/tests/step1/FacetDateGoButton.java

```diff
@@ -0,0 +1,37 @@
+package au.org.emii.portal.tests.step1;
+
+import au.org.emii.portal.tests.BaseTest;
+import org.apache.log4j.Logger;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.util.concurrent.TimeUnit;
+
+public class FacetDateGoButton extends BaseTest {
+
+    private static Logger log = Logger.getLogger(FacetDateGoButton.class.getName());
+
+    @Test
+    public void goButtonTest() {
+        log.info("Loading search page - Step 1");
+        getDriver().get(AODN_PORTAL_SEARCH_PAGE);
+
+        log.info("Selecting Date Facet");
+        portalUtil.selectFacetHeading("Parameter");
+        portalUtil.selectFacetHeading("Platform");
+        portalUtil.selectFacetHeading("Date (UTC)");
+
+        WebElement goButton = portalUtil.getFacetHeadingElement("Date (UTC)").findElement(By.xpath("//button[contains(.,'Go')]"));
```

Comment:
This nested xpath is not obvious by reading the code. But it must be the case or the next line would be rubbish too!